Clarify that table procedures are not supported by FBAC (#24971)
trinodb/trino: docs/src/main/sphinx/security/file-system-access-control.md

(system-file-procedure-rules)=
#### Procedure rules

These rules control the ability of a user to execute procedures using the

> `CALL <catalog>.<schema>.<procedure>(<arguments>)` statement.

Reviewer: Link to docs for it.. 

```suggestion
[CALL <catalog>.<schema>.<procedure>(<arguments>)](/sql/call) statement.
```

Also .. those docs are minimalistic... sigh

Author: But they are still more descriptive than the simple example I gave :) I'll just shorten this to `[CALL](/sql/call)` then.

}
```

(system-file-table-procedure-rules)=
#### Table procedure rules

Table procedures are executed using the

> [ALTER TABLE ... EXECUTE](/sql/alter-table#alter-table-execute) syntax.

Author: Hopefully this is the right way to link this

Reviewer: nope .. you have to introduce an anchor there if you want to link to a specific section..

Author: Eh, not a power-user of MD, so I though that `(alter-table-execute)=` thingy *was* the anchor :)

> [ALTER TABLE ... EXECUTE](alter-table-execute)

But it's in a different file, so how does that work?

Reviewer: Hahah .. reference anchors (and function anchors and so on) in sphinx are global and independent of the filename .. they are all collected before rendering out HTML .. verified and then adjusted to whatever file the reference points to

Reviewer: This is a sphinx thing .. not md .. we just use the myst parser with sphinx .. and myst parser is a markdown extension for various systems.. hth

}
```

(system-file-table-procedure-rules)=
#### Table procedure rules

Table procedures are executed using the following syntax:

```sql
ALTER TABLE <catalog>.<schema>.<table> EXECUTE <procedure>(<arguments>)
```

> File-based access control does not currently support table procedures and

Reviewer: ```suggestion
File-based access control does not support privileges for table procedures and therefore all are effectively allowed.
```

> effectively all are allowed.

Reviewer: ```suggestion

```
